Automattic/pocket-casts-android
# Migrate ShiftyCustomAudio from Java to Kotlin

## Description

Just a simple migration of Java code outlined in #44. This PR migrate [ShiftyCustomAudio.java](https://github.com/Automattic/pocket-casts-android/blob/61fe9bf02411dbb158ec838d2fc4834099e99a4b/modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/ShiftyCustomAudio.java). There's not much else going on here.

## Testing Instructions

1. Play a video podcast like [TED](https://pca.st/podcast/14282660-0258-012e-f994-00163e1b201c).
2. Tap on the bottom left icon to open playback effects.
3. Enable `Trim Silence` and set it to `Mad Max`.
4. Change `Speed` to `2.0x`.
5. Toggle `Volume Boost` and make sure that you can observe volume changes.
6. Let an episode to play for a while.
7. Go to `Profile -> Stats`. Notice there is a change in time save due to `Variable Speed` and `Trim Silence`.

## Checklist
 
#### I have tested any UI changes...
